test/lib/Configuration.test.js

```diff
@@ -244,14 +244,14 @@ describe('lib.Configuration', () => {
 
       assert.equal(config.get('foo.bar.new.path'), 'test')
     })
-    it.skip('should throw an error when attempting to set a value after frozen', () => {
+    it('should throw an error when attempting to set a value after frozen', () => {
       const config = new lib.Configuration(_.cloneDeep(testConfig), { NODE_ENV: 'test' })
       config.freeze()
 
-      //assert.throws(() => config.customObject = 'test', IllegalAccessError)
-      //assert.throws(() => config.customObject.newValue = 'test', IllegalAccessError)
-      assert.throws(() => config.customObject.string = 'b', IllegalAccessError)
-      assert.throws(() => config.set('customObject.string', 'b'), IllegalAccessError)
+      assert.throws(() => config.set('customObject.string', 'b'), lib.Errors.IllegalAccessError)
+      // TODO re-enable
```

Comment:
Did you forget this one ? :)

Reply:
lol, naw, but for some reason I'm having a hard time getting the non virtual setters (computed properties) to freeze.